types/express-graphql/index.d.ts

```diff
@@ -46,11 +53,36 @@ declare namespace graphqlHTTP {
          */
         pretty?: boolean | null;
 
+        /**
+         * An optional function which will be used to validate instead of default `validate`
+         * from `graphql-js`.
+         */
+        customValidateFn?: (
+            schema: GraphQLSchema,
+            documentAST: DocumentNode,
+            rules?: ReadonlyArray<any>,
```

Comment:
As discussed [here](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html), it's not recommended to have optional parameters in callbacks.

Also, the flow types indicate the parameter is not optional (maybe for the same reason as our recommend above).